### ISSUE TITLE
Chapter Transition always shows up for single-page chapters in webtoon/continuous vertical modes

### ISSUE THREAD

**Issue by laundmo** [tachiyomiorg/tachiyomi#9744]
### Steps to reproduce

1. download many pages of a single-page-per-chapter comic (i used Kill Six Billion Demons and Aurora to test(
2. open the first page in either webtoon of continuous vertical mode
3. make sure "always show chapter transition" is disabled
4. scroll down

### Expected behavior

It should seemlessly combin the pages into a vertical fashion

### Actual behavior

it adds a chapter transition page between each chapter (= each page) even though a lot of future chapters are already downloaded 

### Tachiyomi version

0.14.6

### Android version

android 10

### Device

Ulefon Armor

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/help/faq/) and [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by Seishirou101** [comment]
Unsure if this is the intended behavior as this was fixed and webtoon behavior was explicitly left out? #6998

**Comment by arkon** [comment]
Right, #3079 shouldn't have been closed since it still applies to `WebtoonViewer`, but we can track it with this issue instead now.
